main.py

```diff
@@ -85,6 +86,8 @@ def predicate(interaction: Interaction):
 @bot.event
 async def on_ready():
     print(f"{bot.user} has connected to Discord!")
+    bot.server_embed = await tb_embeds.ServersEmbed.create(bot.get_channel(1099805791487266976))
```

Comment:
Including the embed generator in `on_ready` may cause an embed to be generated multiple times, as `on_ready` may be repeatedly called during runtime.
Also, **running this gave me the following exception.**

```Traceback (most recent call last):
  File "C:\Users\admin\PycharmProject\towerbot\venv\Lib\site-packages\discord\client.py", line 441, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\admin\PycharmProject\towerbot\main.py", line 89, in on_ready
    bot.server_embed = await tb_embeds.ServersEmbed.create(bot.get_channel(1099805791487266976))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\PycharmProject\towerbot\tb_embeds.py", line 30, in create
    self.message = await channel.send(embed=self)
                         ^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'send'

Reply:
The bot could delete its own previous messages in the server-info channel; this would resolve the issue of it spitting out multiple embeds